com.unity.shadergraph/Editor/Data/Nodes/Utility/SubGraphNode.cs

```diff
@@ -220,6 +226,16 @@ public void GenerateNodeCode(ShaderStringBuilder sb, GenerationMode generationMo
                 arguments.Add(GetSlotValue(inSlotId, generationMode, prop.concretePrecision));
             }
 
+            var dropdowns = asset.dropdowns;
+            foreach (var dropdown in dropdowns)
+            {
+                var name = GetDropdownEntryName(dropdown.referenceName);
+                if (dropdown.ContainsEntry(name))
+                    arguments.Add(dropdown.IndexOf(name).ToString());
+                else
+                    arguments.Add(dropdown.value.ToString());
```

Comment:
This would happen if you had a dropdown selection in a parent graph, and then deleted the entry from the subgraph.

E.g. 
Subgraph dropdown
A, B, C

Parent graph picks B, and serialize.

Now delete B from subgraph, and save asset.

Now parent graph selection now longer exists, so just goes back to default value of dropdown.